platform/android/audio_driver_opensl.cpp

```diff
@@ -301,15 +315,11 @@ AudioDriver::SpeakerMode AudioDriverOpenSL::get_speaker_mode() const {
 }
 
 void AudioDriverOpenSL::lock() {
-	if (active) {
-		mutex.lock();
-	}
+	output_mutex.lock();
```

Comment:
`if (active)` is not necessary and could create issues(?), `AudioDriverCoreAudio` doesn't check for it